spec/index.html

```diff
@@ -73,23 +73,14 @@
 </section>
 
 <section id="sotd">
-  <p>This document is an edited version of the 2004 RDF XML Syntax
-    Specification Recommendation. The purpose of this revision is
-    to make this
-    document available as part of the RDF 1.2 document set. Changes are
-    limited to revised references, terminology updates, and adaptations to
-    the introduction.
-    The technical content of the document is unchanged, except for
-    the fact that the datatype XMLLiiteral is marked as
-    non-normative in RDF 1.2. The (non-normative) algorithm for
-    parsing XMLLiteral
-    (<a href="#parseTypeLiteralPropertyElt">Sec. 7.2.17</a>)
-    has been updated to be in line with
-    the current state of XML technology. Details of the changes
-    are listed in the <a href="#changes-rdf11">Changes</a>
-    section. Since the edits to this document do not invalidate
-    previous implementations the Director decided no new implementation report was required. </p>
-
+  <p>This document is a part of the RDF 1.2 document suite. The
+    document defines the RDF/XML syntax,
+    a concrete syntax for RDF [[RDF12-CONCEPTS]].</p>
+
+  <p>There have been no substantive changes to this document since [[RDF11-XML]].
```

Comment:
```suggestion
  <p>There have been no substantive changes to this document since [[rdf-syntax-grammar]].
```
There weren't substantive changes for RDF 1.1 either : https://www.w3.org/TR/rdf-syntax-grammar/#h2_sotd